dm/quick-start-with-dm.md

```diff
@@ -1,182 +1,465 @@
+    ```yaml
+    source-id: "mysql-01"
+    from:
+      host: "127.0.0.1"
+      user: "tidb-dm"
+      password: "MyPassw0rd!"    # In production environments, it is recommended to use a password encrypted with dmctl.
+      port: 3306
+    ```
 
-{{< copyable "sql" >}}
+2. Create a DM data source:
 
-```sql
-drop database if exists `testdm`;
-create database `testdm`;
-use `testdm`;
-create table t1 (id bigint, uid int, name varchar(80), info varchar(100), primary key (`id`), unique key(`uid`)) DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin;
-create table t2 (id bigint, uid int, name varchar(80), info varchar(100), primary key (`id`), unique key(`uid`)) DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin;
-insert into t1 (id, uid, name) values (1, 10001, 'Gabriel García Márquez'), (2, 10002, 'Cien años de soledad');
-insert into t2 (id, uid, name) values (3, 20001, 'José Arcadio Buendía'), (4, 20002, 'Úrsula Iguarán'), (5, 20003, 'José Arcadio');
-```
+    ```shell
+    tiup dmctl --master-addr 127.0.0.1:8261 operate-source create mysql-01.yaml
+    ```
 
-## Step 5: Create a data migration task
+## Step 4: Create a TiDB DM task
 
-1. Create a task configuration file `testdm-task.yaml`:
+After configuring the source database, you can create a migration task in TiDB DM. This task references the source MySQL instance and defines the connection details for the target TiDB database.
 
-    {{< copyable "" >}}
+1. Create a DM task configuration file `tiup-playground-task.yaml`:
 
     ```yaml
-    name: testdm
-    task-mode: all
+    # Task
+    name: tiup-playground-task
+    task-mode: "all"              # Execute all phases - full data migration and incremental sync.
+
+    # Source (MySQL)
+    mysql-instances:
+      - source-id: "mysql-01"
 
+    ## Target (TiDB)
     target-database:
       host: "127.0.0.1"
       port: 4000
       user: "root"
-      password: "" # If the password is not empty, it is recommended to use a password encrypted with dmctl.
+      password: ""                # If the password is not empty, it is recommended to use a password encrypted with dmctl.
+    ```
 
-    # Configure the information of one or multiple data sources
-    mysql-instances:
-      - source-id: "mysql-01"
-        block-allow-list:  "ba-rule1"
+2. Start the task using the configuration file:
+
+    ```shell
+    tiup dmctl --master-addr 127.0.0.1:8261 start-task tiup-playground-task.yaml
+    ```
+
+## Step 5: Verify the data replication
+
+After starting the migration task, verify whether data replication is working as expected. Use the `dmctl` tool to check the task status, and connect to the target TiDB database to confirm that the data has been successfully replicated from the source MySQL database.
+
+1. Check the status of the TiDB DM task:
+
+    ```shell
+    tiup dmctl --master-addr 127.0.0.1:8261 query-status
+    ```
+
+2. Connect to the TiDB target database:
+
+    ```shell
+    mysql -uroot -h127.0.0.1 -P4000 --prompt 'tidb> '
```

Comment:
```suggestion
    mysql --host 127.0.0.1 --port 4000 -u root --prompt 'tidb> '
```
- Use long-form flags `--host` and `--port` to improve readability and avoid confusion between similar flags like `-p` (password) and `-P` (port).
- The flag order matches the standard output from `tiup playground`, maintaining consistency throughout our documentation:

https://github.com/pingcap/docs/blob/cc9fd97598e114bdeb6d24d476eafba2792b22a7/dm/quick-start-with-dm.md?plain=1#L48-L50

Reply:
@alastori FYI.